From 40fe99a372df5297d3b19588875b9f2802fe00e6 Mon Sep 17 00:00:00 2001
From: AndersBroman <a.broman@bredband.net>
Date: Fri, 28 Mar 2014 06:45:01 +0100
Subject: [PATCH 2/2] Deleted one line to much in previous chgange.

Change-Id: I19d1153558130356286bac55f6cb321d30a767b1
Reviewed-on: https://code.wireshark.org/review/852
Reviewed-by: Anders Broman <a.broman58@gmail.com>
(cherry picked from commit b2509f67b8d64ecb5de902b9954c7659605c4395)
Reviewed-on: https://code.wireshark.org/review/1638
---
 ui/gtk/graph_analysis.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ui/gtk/graph_analysis.c b/ui/gtk/graph_analysis.c
index d181b23..0ee7abf 100644
--- a/ui/gtk/graph_analysis.c
+++ b/ui/gtk/graph_analysis.c
@@ -825,6 +825,7 @@ static void dialog_graph_draw(graph_analysis_data_t *user_data)
 	if ( GDK_IS_DRAWABLE(user_data->dlg.pixmap_time) ) {
 		cr = gdk_cairo_create (user_data->dlg.pixmap_time);
 		gdk_cairo_set_source_rgba (cr, &bg_color);
+		cairo_rectangle (cr, 0, 0, draw_area_time_alloc.width, top_y_border);
 		cairo_fill (cr);
 		cairo_destroy (cr);
 
-- 
1.9.1

